resources/js/processes-catalogue/components/utils/EmptySearch.vue

```diff
@@ -1,24 +1,43 @@
 <template>
-  <div>
+  <div class="py-5">
     <div class="d-flex justify-content-center my-5">
-      <i class="empty-processes fa fa-umbrella-beach" />
+      <img
+          class="image d-flex empty-processes"
+          src="/img/launchpad-images/emptyResults.svg"
+          alt="Empty Results"
+        >
     </div>
-    <p class="text-center message-empty">
+    <p class="text-center title-empty">
       {{ $t("No Processes Available") }}
```

Comment:
Please update this label according FIGMA:
`No results to show`
Please consider the translations too